### ISSUE TITLE
parallel benchmark

### ISSUE THREAD

**Issue by DmitryDMR** [nicehash/NiceHashMiner#768]
i run benchmarks for rx 580 and rx vega in parallel mode.
and result for rx vega applayed to the rx 580, but result for rx vega applayed to the rx 580.

**Comment by S74nk0** [comment]
Issue related to old nhm version/build. Please use latest version and open a new issue for bugs and/or features.
